src/firmware/app/control/python_controller/generate_difference_equation.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+import sys
+import control as ct
+from control.matlab import *
+
+CONTROLLER_DIFFERENCE_EQUATION_COEFFICIENTS = """
+#define WHEEL_SPEED_CONTROLLER_TICK_TIME 1e7
```

Comment:
Where does 1e7 come from? What are the units?

Reply:
`WHEEL_SPEED_CONTROLLER_TICK_TIME` is the period we expect the controller to run in. 1e7 was just taken from the sample snippet @garethellis0 posted above. Should be in seconds, will add a comment clarifying that.

Reply:
So this is an example for now, or a reasonably "real" value? Either way, seems maybe like something the python controller or something should define rather than being "hidden" in the generated code template? @garethellis0 @EvanMorcom 

Also I'm assuming that should be `1e-7` then?

Reply:
I could have `generate` take in a parameter and generate it instead.

Reply:
@garethellis0 @EvanMorcom bumping this

Reply:
@EvanMorcom - is there a function that @kelvinkoon can use from the model to get the tick time it was based on?

Reply:
@EvanMorcom bumping this

Reply:
Ah yes. There is a member if the discrete TF class in the python control toolbox called 'dt' that gives this period. 
<img width="338" alt="Screen Shot 2020-07-02 at 10 03 32 AM" src="https://user-images.githubusercontent.com/25271038/86389542-4b4b0c00-bc4b-11ea-95d6-6825c55e68d2.png">

Reply:
Added.